tests/auto/prompt_scorer.test.py

```
@@ -0,0 +1,497 @@
+from datetime import datetime
+
+# Import the classes to test
+import sys
+sys.path.append('../../beta-prompts')
+from beta_prompts.prompt_scorer_02 import PromptScorer, TaskMetrics
+
+
+class TestTaskMetrics:
+    """Test TaskMetrics dataclass"""
+    
+    def test_task_metrics_creation(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_task_metrics_creation`

Add return type annotation: `None`

(ANN201)